### DISCUSSION TITLE
15.3.7

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#724] [General]
## Releases Notes for 15.3.7
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v15.3.7/draw.io-15.3.7-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v15.3.7/draw.io-15.3.7-windows-no-installer.exe)
[macOS - Intel](https://github.com/jgraph/drawio-desktop/releases/download/v15.3.7/draw.io-x64-15.3.7.dmg)
[macOS - ARM](https://github.com/jgraph/drawio-desktop/releases/download/v15.3.7/draw.io-arm64-15.3.7.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v15.1.3/drawio-amd64-15.1.3.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v15.1.3/drawio-x86_64-15.1.3AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v15.1.3/drawio-x86_64-15.1.3.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Reverts from Electron 15 to Electron 14. There were stability problem we suspect are caused by 15.
- We are fixing Linux releases at 15.1.3 build for now. There have been problems with the native picker since 15.1.3.
- Updates to [diagrams.net core 15.3.7](https://github.com/jgraph/drawio/blob/v15.3.7/ChangeLog). All changes from 15.1.4 to 15.3.7, inclusive, are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v15.3.7'>15.3.7</a>.</em>

**Comment by iGadget** [comment]
Great to see another release. However, the Linux links all point to v15.1.3. Has their build failed? I don't see them in the assets list...
EDIT - never mind, I should have read the ChangeLog. Hope the issues get sorted out soon.

**Reply by davidjgraph** [reply]
https://github.com/electron/electron/issues/31152 is affecting Linux builds using Electron 14 and later. 15.1.3 used Electron 12, as I remember, so that's the stable Linux build for now.

If there's no attempt to fix 31152, we'll do a Linux only build using Electron 13.4.0 so there's an updated version.

**Comment by altwohill** [comment]
The auto-updater isn't working in Windows - getting a 404 error

![image](https://user-images.githubusercontent.com/629254/135171612-ca3cd090-278e-4422-ab55-2e12991838e4.png)

**Comment by janis-j2b** [comment]
`deb` package pointing to older version v15.1.3, actually causing downgrade...

**Reply by NRGLine4Sec** [reply]
same issue for AppImage version

**Comment by hlldz** [comment]
I'm getting a checksum mismatch error for Mac OS X.

> Error occurred while fetching updates. sha512 checksum mismatch, expected tiQzegFd6QQjjm5iR4V+Nxs4BLAR9+tKVt0k/Fkcq+IEF/T2ADyJNXgMPpMgp5zV6m8Qzum5WVmU54B8Li+XDQ==, got sYjeZ7w7A4FvzmYFmn718FVWmBw3MSrqew/rvxK7VVDaWCVVLg9gn5wsi+v7OTL/3VBwbmdZra5vA8+3Qt7atQ==
